src/main/java/me/rayzr522/jsonmessage/JSONMessage.java

```diff
@@ -205,7 +202,29 @@ public JSONMessage color(ChatColor color) {
         last().setColor(color);
         return this;
     }
-
+    
+    /**
+     * Sets the color of the current message part.
+     * <br>This method allows you to provide a hex color, which can be used in 1.16 to color the message in any Hex color
+     * imaginable.
+     * 
+     * @param color The color to set
+     * @return This {@link JSONMessage} instance
+     */
+    public JSONMessage color(String color) {
+        if(color.startsWith("#") && ReflectionHelper.MAJOR_VER < 16) {
```

Comment:
consistency, there should be spaces between all if statements and their parenthesis (noticed this in a few places, should be easy to fix by tweaking your IDE formatter profile and reformatting tho)